static/app/views/issueDetails/traceTimeline/traceTimelineEvents.tsx

```diff
@@ -133,13 +134,13 @@ function NodeGroup({
   event: Event;
   timeRange: [number, number];
 }) {
-  const totalSubColumns = Math.floor(columnSize / subWidth);
+  const totalSubColumns = Math.floor(columnSize / CHILD_WIDTH);
   const durationMs = timeRange[1] - timeRange[0];
   const eventsByColumn = getEventsByColumn(
     durationMs,
     colEvents,
     totalSubColumns,
-    timeRange[1]
+    timeRange[0]
```

Comment:
this ended up being the main issue, was using the end date which was causing the column to be a negative number since `event - endtime` is negative